app/assess/routes.py

```diff
@@ -297,6 +299,9 @@ def landing():
 
     search_params = {
         "search_term": "",
+        "search_in": "project_name,application_short_id",
```

Comment:
Just wondering whether this should be a store concern, rather than letting the frontend decide which fields to query?

Reply:
We thought a frontend thing as the front end will be telling users which fields the search applies to, so rather than duplicate that info it's all in one service.

The frontend search box has the label 'Search in project name and ref' or similar.